docs/fundamentals/jcloud/autoscale.md

```diff
@@ -0,0 +1,55 @@
+# Autoscale
```

Comment:
Should we rename this to `Autoscaling` ?

Reply:
Yea that makes sense, `Knative` uses the same.